pootle/apps/pootle_store/models.py

```diff
@@ -1280,16 +1280,27 @@ def create(self, *args, **kwargs):
         if "filetype" not in kwargs:
             filetypes = ProjectFiletypes(kwargs["translation_project"].project)
             kwargs['filetype'] = filetypes.choose_filetype(kwargs["name"])
+        tp = kwargs["translation_project"]
+        if tp.language == tp.project.source_language:
```

Comment:
in particular, im not clear what happens throughout the system if you were to set a source_language (ie not templates) and then also add a templates tp